source/cosmosdb-support.txt

```diff
@@ -0,0 +1,75 @@
+.. _cosmosdb-compat:
+
+:noprevnext:
+
+=============================
+Azure Cosmos DB Compatibility
+=============================
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta:: 
+   :keywords: microsoft, support
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+.. figure:: /figures/cosmosdb.png
+   :alt: Azure Cosmos DB logo
+
+`Azure Cosmos DB for MongoDB <https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/introduction>`__
+makes it easy to use Azure Cosmos DB as if it were a MongoDB database.
+With Azure Cosmos DB, you can run the same application code and use the same drivers
+and tools that you use with MongoDB.
+
+Though Azure Cosmos DB for MongoDB implements MongoDB's
+`Wire Protocol <https://www.mongodb.com/docs/manual/reference/mongodb-wire-protocol/>`__
+to allow MongoDB's `drivers <https://www.mongodb.com/docs/drivers/>`__ to connect and
+interact with Cosmos DB as though it were a MongoDB host, this
+implementation has limitations.
+
+Version Information
+-------------------
+
+Azure Cosmos DB supports MongoDB v4.2, or MongoDB v5.0 for vCore clusters.
+These versions don't support all the newer features and functionality in the versions of
+MongoDB available on `MongoDB Atlas. <https://www.mongodb.com/docs/atlas/>`__
+
+.. tip:: MongoDB Version History
+   
+   To learn more about the features available in each version of MongoDB, see 
+   `MongoDB Evolved - Version History. <https://www.mongodb.com/evolved>`__
+
+Compatibility
+-------------
+
+As of October 2023, Azure Cosmos DB is about 32 percent compatible with the
+MongoDB API.
+
+On Azure Cosmos DB for MongoDB v4.2, the following MongoDB v4.2 features are
+available in a limited capacity or are not supported at all:
+
+- **Features**: `Text indexes <https://www.mongodb.com/docs/manual/core/indexes/index-types/index-text/>`__,
+  `2d indexes <https://www.mongodb.com/docs/manual/core/indexes/index-types/geospatial/2d/#std-label-2d-index>`__,
+  `hashed indexes <https://www.mongodb.com/docs/manual/core/indexes/index-types/index-hashed/#std-label-index-type-hashed>`__,
+  `case-insensitive indexes <https://www.mongodb.com/docs/manual/core/index-case-insensitive/>`__,
+  `sparse indexes <https://www.mongodb.com/docs/manual/core/index-sparse/>`__
+- **Aggregation Stages**: ``$collStats``, ``$bucket``, ``$bucketAuto``, ``$currentOp``,
+  ``$indexStats``, ``$listLocalSessions``, ``$listSessions``, ``$lookup``
+
+To learn more about Azure Cosmos DB's compatibility with MongoDB v4.2,
+see Microsoft's `Azure Cosmos DB for MongoDB (4.2 server version): Supported features and syntax <https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/feature-support-42>`__
+documentation.
+
+For the most current compatibility status between Azure Cosmos DB and MongoDB, see
+`Is Cosmos DB Compatible with MongoDB Atlas? <https://www.iscosmosdbreallymongodb.com/>`__
```

Comment:
S: to make the link target more explicit and avoid ending this sentence with the link question mark, you could reword to something like this: 

```suggestion
the `Is Cosmos DB Compatible with MongoDB Atlas? <https://www.iscosmosdbreallymongodb.com/>`__ website.
```